src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -150,13 +150,13 @@ public static string DownloadImages()
         /// </summary>
         /// <remarks>
         /// The model is downloaded from
-        /// https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model
+        /// https://github.com/dotnet/machinelearning-testdata/blob/296625f4e49d50fcd6a48a0d92bea7584e198c0f/Microsoft.ML.TensorFlow.TestModels/sentiment_model
         /// The model is in 'SavedModel' format. For further explanation on how was the `sentiment_model` created
-        /// c.f. https://github.com/dotnet/machinelearning-testdata/blob/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model/README.md
+        /// c.f. https://github.com/dotnet/machinelearning-testdata/blob/296625f4e49d50fcd6a48a0d92bea7584e198c0f/Microsoft.ML.TensorFlow.TestModels/sentiment_model/README.md
         /// </remarks>
         public static string DownloadTensorFlowSentimentModel()
         {
-            string remotePath = "https://github.com/dotnet/machinelearning-testdata/raw/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model/";
+            string remotePath = "https://github.com/dotnet/machinelearning-testdata/raw/296625f4e49d50fcd6a48a0d92bea7584e198c0f/Microsoft.ML.TensorFlow.TestModels/sentiment_model/";
```

Comment:
_(tagged this on a line of code to allow for conversation)_

> harishsk commented 22 seconds ago
Sorry, I am not sure I follow. Can you please explain what you mean by "Otherwise older versions of the code will break when the repo is reorganized, or the files renamed." ?

The code here grabs a file from https://github.com/dotnet/machinelearning-testdata/raw/master/Microsoft.ML.TensorFlow.TestModels/sentiment_model/imdb_word_index.csv. 

This is a link to master in another repo (dotnet/machinelearning-testdata). 

Lets say in year, someone is still using the current version of ML.NET. When that version grabs the linked file, it will still use that same URL. But in the mean time, the master branch of that repo will have changed, and perhaps breaking that link. For instance if they decide to re-organize the datasets in the repo, the URL will no longer work.

Now if we link to a specific commit in that repo, the changes to master (eg. them reorganizing the datasets in the repo) will not break the commit-pinned link this PR uses.

Reply:
Thanks! That make sense.